### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by elbotho**

### CODE REVIEW

**Comment by Entkenntnis** [comment]
Could it be that we are losing some translations here?

**Comment by elbotho** [comment]
> Could it be that we are losing some translations here?

@Entkenntnis because of the conflict, or did you see something else?

File: src/data/es/index.ts [hunk 1]
```diff
@@ -105,11 +110,6 @@ export const instanceData = {
       applet: 'Load Applet from %provider%',
       twingle: 'Load Donation Form'
     },
-    notifications: {
-      notifications: "Notificaciones",
-      pleaseLogInLink: "Por favor, inicia sesión",
-      pleaseLogInText: "para ver tus notificaciones."
```
Comment (Entkenntnis) [inline src/data/es/index.ts:111]: For example: This three translations will be lost?

(looks like we have to do some manual work if we want to move translations around ...)

Comment (elbotho) [reply src/data/es/index.ts:111]: Ah, ok. That is intended. The guard component now has more general strings, so we don't have to add new strings for every function.
I will solve the conflicts and merge.

**Pull request merged**